scripts/lcoe.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
+
+
+def calc_lcoe(capacity=1, power_yield=1, distance =1, depth =1, af=1):
+
+    tech = pd.read_csv("data/lcoe/tech_data.csv", index_col=0)
+
+    capacity = turbine.capacity ## get from energyharvest
+    power_yield = timeseries.sum()  # get from energyharvest
+    distance = get_distance_to_coast(self)  # get from this function
+    depth = water_depth(depth_map=depth_map, depth=depth_map[row, col]) # get frpm this function
```

Comment:
Die Funktionen musst du importieren. 
get_distance_to_coast ist eine funktion der klasse location. Das ist wie in pandas der DataFrame ja auch verschiedene Funktionen hat, wie zum beispiel .sum() verschiedene Summen bilden kann. Diese Funktion (.sum()) beuzieht sich dann auf die Instanz der Klasse (self). Also muss für diese funktionen zuerst eine Location entweder übergeben werden oder angelegt werden aus irgendwas. Es ist aber auch vollkommen in Ordnung das (wie du es gemacht hast) im Funktionsaufruf zu erledigen und einfach zu sagen du bekommst diese Zahlen von Extern.
Frag gerne nochmal nach dazu, schwierig hier zu erklären.

Reply:
Dann lass da gern am Dienstag nochmal zusammen drauf schauen :)